validation-test/Sema/SwiftUI/rdar58972627.swift

```diff
@@ -0,0 +1,40 @@
+// RUN: %target-swift-frontend %s -target x86_64-apple-macosx10.15 -emit-sil -verify
+// REQUIRES: objc_interop
+// REQUIRES: OS=macosx
+
+import SwiftUI
```

Comment:
Can we exercise this without importing SwiftUI?

Reply:
I couldn't reproduce this problem in any other way because `ExprRewriter` does the right thing and we have tests for that, so I've added SwiftUI specific example to make sure that it works because function transformation does its own thing...